api/DOMTokenList.json

```diff
@@ -93,6 +93,54 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "add_multiple_arguments": {
+          "__compat": {
+            "description": "Multiple arguments for <code>add()</code>",
+            "support": {
+              "chrome": {
+                "version_added": "25"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "26"
+              },
+              "firefox_android": {
+                "version_added": "26"
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": "15"
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": "7"
+              },
+              "safari_ios": {
+                "version_added": "7"
+              },
+              "samsunginternet_android": {
+                "version_added": true
```

Comment:
This should be “1.5”, the Samsung Internet release based off of Chrome Android 25.